.travis.yml

```diff
@@ -38,7 +38,7 @@ script:
  - set -o pipefail
  - programs/build_helpers/buildstep run.chain_test 240 "libraries/fc/tests/run-parallel-tests.sh tests/chain_test"
  - programs/build_helpers/buildstep run.cli_test 60 "libraries/fc/tests/run-parallel-tests.sh tests/cli_test"
- - 'programs/build_helpers/buildstep prepare.sonar 20 "find libraries/[acdenptuw]*/CMakeFiles/*.dir programs/[cdgjsw]*/CMakeFiles/*.dir -type d | while read d; do gcov -o \"\$d\" \"\${d/CMakeFiles*.dir//}\"/*.cpp; done >/dev/null; programs/build_helpers/set_sonar_branch sonar-project.properties" || true'
- - 'programs/build_helpers/buildstep run.sonar 1200 "which sonar-scanner && sonar-scanner" || true'
+# - 'programs/build_helpers/buildstep prepare.sonar 20 "find libraries/[acdenptuw]*/CMakeFiles/*.dir programs/[cdgjsw]*/CMakeFiles/*.dir -type d | while read d; do gcov -o \"\$d\" \"\${d/CMakeFiles*.dir//}\"/*.cpp; done >/dev/null; programs/build_helpers/set_sonar_branch sonar-project.properties" || true'
+# - 'programs/build_helpers/buildstep run.sonar 1200 "which sonar-scanner && sonar-scanner" || true'
```

Comment:
No need to disable this. We use ccache, it means for every new build, the first time could fail anyway, but some cache will be stored, then we restart travis, due to the cache, it would need much less time to build, so we'll have the time to run the sonar stuff. See https://github.com/bitshares/bitshares-core/pull/1264.

Reply:
I don't think it has enough time with my code included, but I'll poke it and we'll see.

Reply:
@pmconrad does it make sense to add a `- ccache -s` before ` - set -o pipefail`?

Reply:
Looks like it still doesn't have enough time.

Reply:
Alright, so I removed just the restriction predicate code from coverage analysis, and now it passes :)